src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -589,13 +591,13 @@ <h3 class="panel-title">{editing, select, 1 {Edit} other {Add}} user</h3>
         <div class="button-group text-right">
           <cd-submit-button type="button"
                             (submitAction)="onSubmit()"
-                            [form]="userForm">
-            <ng-container i18n>{editing, select, 1 {Update} other {Add}}</ng-container>
-          </cd-submit-button>
+                            [form]="userForm"i18n="form action button|Example: Create Pool@@formActionButton"
```

Comment:
```suggestion
                            [form]="userForm"
                            i18n="form action button|Example: Create Pool@@formActionButton"
```

Reply:
Great catch! This passed all the linters and builders unnoticed :open_mouth:

Reply:
The HTML linter we use is not that complete, so lots of stuff is still only caught manually